docs/rst/recording/usage/configuration.rst

```diff
@@ -339,10 +339,46 @@ The recorder output file does support the following configuration settings under
         - ``unsigned int``
         - ``0``
 
+    *   - Resource limits
+        - ``resource-limits``
+        - :ref:`recorder_usage_configuration_resource_limits`
+        - ``list``
```

Comment:
Not really a list, a map in any case. But I would better put the link to the section instead.